test/tdigest_test.rb

```diff
@@ -135,6 +137,13 @@ def test_that_it_has_a_version_number
         123829787.23404256,
         103191489.36170213]
     end
+
+    it 'does not blow up if data comes in sorted' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.